docs/source/index.rst

```diff
@@ -0,0 +1,38 @@
+.. Lusidtools documentation master file, created by
+   sphinx-quickstart on Tue Mar 31 10:24:09 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Lusidtools's documentation!
+======================================
+
+.. toctree::
+   :maxdepth: 2
+   lusidtools/cocoon/cocoon
+   lusidtools/cocoon_printer
+   lsuditools/utilities
```

Comment:
Can you please add 
lusidtools/pandas_utils
lusidtools/extract

Reply:
Added them